packages/evm/src/evm.ts

```diff
@@ -767,7 +767,10 @@ export class EVM implements EVMInterface {
     const err = result.execResult.exceptionError
     // This clause captures any error which happened during execution
     // If that is the case, then all refunds are forfeited
-    if (err) {
+    // There is one exception: if the CODESTORE_OUT_OF_GAS error is thrown
+    // (this only happens the Frontier/Chainstart fork)
+    // then the error is dismissed
+    if (err && err.error !== ERROR.CODESTORE_OUT_OF_GAS) {
```

Comment:
We can do this as a sanity check, but this `CODESTORE_OUT_OF_GAS` error can /only/ happen in Frontier/Chainstart. If you want I can also add it as unit test?

Reply:
I understand your concern but I did not add it, because `CODESTORE_OUT_OF_GAS` can /only/ happen in Frontier/Chainstart, see; https://github.com/ethereumjs/ethereumjs-monorepo/blob/bd9735f0dd0806bb4778f1af14ae574d7540a895/packages/evm/src/evm.ts#L575 (I explicitly checked this). It is the only line where this error is thrown, and as you can see there is an if statement above which checks if we are `>= Homestead` (`Homestead` was the first fork of Ethereum, so the fork after Chainstart/Frontier)

Reply:
I'm good with that. Will approve.